packages/labs/analyzer/src/lib/model.ts

```diff
@@ -434,35 +440,51 @@ export class ClassDeclaration extends Declaration {
   }
 
   /**
-   * Returns iterator of the `ClassField`s defined on the immediate class
-   * (excluding any inherited members).
+   * Returns iterator of the non-static `ClassField`s defined on the immediate
+   * class (excluding any inherited members).
    */
-  get fields() {
+  get fields(): IterableIterator<ClassField> {
     return this._fieldMap.values();
   }
 
   /**
-   * Returns iterator of the `ClassMethod`s defined on the immediate class
+   * Returns iterator of the static `ClassField`s defined on the immediate class
    * (excluding any inherited members).
    */
+  get staticFields(): IterableIterator<ClassField> {
+    return this._staticFieldMap.values();
+  }
+
+  /**
+   * Returns iterator of the non-static `ClassMethod`s defined on the immediate
+   * class (excluding any inherited members).
+   */
   get methods(): IterableIterator<ClassMethod> {
     return this._methodMap.values();
   }
 
+  /**
+   * Returns iterator of the static `ClassMethod`s defined on the immediate
+   * class (excluding any inherited members).
+   */
+  get staticMethods(): IterableIterator<ClassMethod> {
+    return this._staticMethodMap.values();
+  }
+
   /**
    * Returns a `ClassField` model the given name defined on the immediate class
    * (excluding any inherited members).
    */
-  getField(name: string): ClassField | undefined {
-    return this._fieldMap.get(name);
+  getField(name: string, isStatic = false): ClassField | undefined {
```

Comment:
Given that we have `fields` and `staticFields`, I'm not sure what value there is in a parameter to separate the two here. I would default to mirroring the two APIs and making `getStaticField()`, etc.

Reply:
Sgtm, done